assets/scss/hero.scss

```diff
@@ -29,61 +53,85 @@
   }
 
   &.browse-jumbotron {
-    .container {
-      padding-bottom: 3rem;
+    margin-top: 2rem;
+    border-radius: 0.5rem;
+    overflow: hidden;
+    &:before {
+      display: none;
     }
-    &:after {
-      border-bottom: 145px solid $bodygrey;
-      border-left: 60px solid transparent;
-      bottom: 0;
-      content: '';
-      display: block;
-      height: 0;
-      position: absolute;
-      right: 0;
-      width: 0;
-      z-index: 3;
+    .container {
+      .card {
+        margin: 1rem;
+        max-width: 37rem;
```

Comment:
The card looks too large, it takes over 50% of the banner. Is that correct?

Reply:
ok I was using rem based on the design px but now I've limited to 50%